Assets/Scripts/Enemies/SentryTrigger.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SentryTrigger : MonoBehaviour
+{
+    private void OnTriggerEnter(Collider collider)
+    {
+        if (collider.gameObject.name == "GameDollyCart")
```

Comment:
This should be made so that it takes a tag rather than the gameObjects name because it does not work at the moment since the player prefab was updated and its named got changed